src/components/CallView/shared/LocalAudioControlButton.vue

```diff
@@ -118,6 +129,9 @@ export default {
 
 	mounted() {
 		subscribe('local-audio-control-button:toggle-audio', this.updateDeviceState)
+		if (this.mediaDefaultStateEnabled) {
+			this.model.disableAudio()
```

Comment:
When opening MediaSettings after the first call, audio/video would be toggled off. ot sure yet, how we should handle that

Maybe force enable for device preview before call and show a hint 'Your camera and mic will be switched off when you join a call' depending on mediaDefaultStateEnabled

Reply:
I am not sure if it's privacy-okay to always force enable it? If user turned on their cam in the meeting, it will be on (following their pref). If they didn't, it will stay off.

Reply:
Just to make sure:
1. I go to Talk settings and turn of my media by default
2. Any call I start shall be "off" with and without the device picker
3. The only place that changes the user preference is the "Talk settings", not any action during a call

Reply:
Yes, it's like that. Maksim is mentioning the state in **media settings** not in call.

Reply:
yep, I meant 'Media settings only';
So users can preview themselves right before the call, but knowing, that camera will be turned off, as soon as they joined

Reply:
Let's leave until we see in real cases. Not a big deal to focus on now